packages/main/src/components/FilterBar/FilterDialog.tsx

```diff
@@ -197,7 +222,61 @@ export const FilterDialog = (props: FilterDialogPropTypes) => {
   const hideValuesText = i18nBundle.getText(HIDE_VALUES);
   const fieldText = i18nBundle.getText(FIELD);
   const fieldsByAttributeText = i18nBundle.getText(FIELDS_BY_ATTRIBUTE);
-  const resetWarningText = i18nBundle.getText(FILTER_DIALOG_RESET_WARNING);
+
+  const visibleChildren = () =>
+    children.filter((item) => {
```

Comment:
```suggestion
    Children.toArray(children).filter((item) => {
```
just to be on the safe side?

Reply:
This will update the `key` of the `children` which will rerender them, it also breaks the toggle filter feature, because we still use a `key` there to identify which child has changed (we should probably revise this at some point).

`.filter` is already save to use here, since the dialog always receives an array of children. I'll adjust the type for this